pkg/kubelet/container/helpers.go

```diff
@@ -423,3 +423,20 @@ func HasAnyRegularContainerStarted(spec *v1.PodSpec, statuses []v1.ContainerStat
 
 	return false
 }
+
+func GetPendingInitContainers(pod *v1.Pod) []v1.Container {
+	startedContainers := map[string]bool{}
```

Comment:
suggestion: use the `k8s.io/apimachinery/pkg/util/sets` package

```suggestion
	startedInitContainers := sets.String{}
```

Reply:
Thanks for suggestion, already changed.